azurerm/internal/services/network/firewall_resource.go

```diff
@@ -467,6 +497,47 @@ func flattenArmFirewallIPConfigurations(input *[]network.AzureFirewallIPConfigur
 	return result
 }
 
+func expandArmFirewallDNSSetting(input []interface{}) map[string]*string {
+	if len(input) == 0 {
+		return nil
+	}
+
+	v := input[0].(map[string]interface{})
+
+	var servers []string
+	for _, server := range v["servers"].([]interface{}) {
+		servers = append(servers, server.(string))
+	}
+
+	return map[string]*string{
+		"Network.DNS.EnableProxy": utils.String(fmt.Sprintf("%t", v["enabled"].(bool))),
+		"Network.DNS.Servers":     utils.String(strings.Join(servers, ",")),
```

Comment:
Can we file a Swagger bug about these? Since this is a dictionary there's no guarantees there won't be breaking changes here

Reply:
Sure, Azure/azure-rest-api-specs#11278

Reply:
could we link to this in the code?

Reply:
Yes, I have added them [in my last commit](https://github.com/terraform-providers/terraform-provider-azurerm/pull/8878/commits/de7d098fb7c8b2548e59622356f927a6f5456224#diff-dff86c50f3e4bb4e7322805d8cf371d2fbd7e58d00444af8e33a4a34e0ec9d01R512).